appveyor.yml

```diff
@@ -25,6 +24,17 @@ test:
     - '**\Sarif.Driver.UnitTests.dll'
     - '**\Sarif.FunctionalTests.dll'
     - '**\Sarif.ValidationTests.dll'
+    
+artifacts:
+  - path: 'bld\bin'
+
+notifications:
+  - provider: Email
+    to:
+      - vsarcplat@microsoft.com
+    subject: 'Sarif-SDK Build {{status}}'
+    message: 'In reference to {{commitID}}. View output <a href="{{buildURL}}">here</a> '
```

Comment:
Yes. Email message still needs to be cleaned up. But it's by adding {{commitMessage}} into a relevant part of the body